docs/getting-started.rst

```diff
@@ -42,12 +42,18 @@ It should be noted that even in most complex logging setups the example would st
 Using the defaults, as above, is equivalent to::
 
    import structlog
-   structlog.configure(processors=[
-       structlog.processors.StackInfoRenderer(),
-       structlog.processors.format_exc_info,
-       structlog.processors.TimeStamper(),
-       structlog.dev.ConsoleRenderer()
-   ])
+   structlog.configure(
+       processors=[
+           structlog.processors.StackInfoRenderer(),
+           structlog.processors.format_exc_info,
+           structlog.processors.TimeStamper(),
+           structlog.dev.ConsoleRenderer()
+       ],
+       wrapper_class=structlog.BoundLogger,
+       context_class=dict,
```

Comment:
could you add a comment here explaining that it's if the runtime has ordered dicts and OrderedDict otherwise?

Reply:
Of course, thanks.